Realm/Tests/SwiftUISyncTestHostUITests/SwiftUISyncTestHostUITests.swift

```diff
@@ -222,18 +180,69 @@ extension SwiftUISyncTestHostUITests {
         syncButtonView.tap()
     }
 
-    func logoutAllUsers() {
+    fileprivate func logoutAllUsers() {
         let loginButton = application.buttons["logout_users_button"]
         XCTAssertTrue(loginButton.waitForExistence(timeout: 2))
         loginButton.tap()
     }
 
-    public func randomString(_ length: Int) -> String {
+    fileprivate func randomString(_ length: Int) -> String {
         let letters = "abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ0123456789"
         return String((0..<length).map { _ in letters.randomElement()! })
     }
 }
 
+class SwiftUISyncTestHostUITests: SwiftUISyncTestCases {
+    override func setUp() {
+        super.setUp()
+        continueAfterFailure = false
+
+        try? FileManager.default.createDirectory(at: clientDataRoot, withIntermediateDirectories: true)
+        try? FileManager.default.createDirectory(at: appClientDataRoot, withIntermediateDirectories: true)
+
+        // Create App once for this Test Suite
+        if SwiftUISyncTestHostUITests.appId == nil {
+            do {
+                let appId = try RealmServer.shared.createApp()
+                SwiftUISyncTestHostUITests.appId = appId
+            } catch {
+                XCTFail("Cannot initialise test without a creating an App on the server")
+            }
+        }
+
+        // Instantiate App from appId after
+        do {
+            app = try getApp()
+        } catch {
+            print("Error creating user \(error)")
+        }
+    }
+
+    private func getApp() throws -> App {
+        // Setup App for Testing
+        let appConfiguration = RLMAppConfiguration(baseURL: "http://localhost:9090",
```

Comment:
Swift code should use the non-RLM-prefixed name.

There's a lot of incidentally duplicated code in each of the subclasses which would be easy to extract.